install_config/persistent_storage/topics/glusterfs_images.adoc

```diff
@@ -0,0 +1,23 @@
+// The GlusterFS variables and associated images
+// Valid only for openshift-enterprise
+// Included in the following files:
+// * install/configuring_inventory_file.adoc
+// * install_config/persistent_storage/persistent_storage_glusterfs.adoc
+
+// Including expanded macro from: _snippets/glusterfs.adoc
+:gluster: Red Hat Gluster Storage
+
+A valid image tag is required for your deployment to succeed. Replace `<tag>`
+with the version of {gluster} compatible with {product-title} {product-version}
+as described in the
+link:https://access.redhat.com/articles/3403951[interoperability matrix] for the
```

Comment:
This link should be changed to point to this page: https://access.redhat.com/articles/3930371

Reply:
@jarrpa, updated, thanks!